telegraf/1.18/alpine/Dockerfile

```diff
@@ -29,6 +29,10 @@ RUN set -ex && \
 
 EXPOSE 8125/udp 8092/udp 8094
 
+RUN addgroup -S telegraf
+RUN adduser -S telegraf -G telegraf
```

Comment:
Does the new telegraf user have access to the files in /etc/telegraf and /etc/telegraf.d? I'm not sure what ownership the files have after being untarred.